src/pocketmine/Server.php

```diff
@@ -2686,6 +2687,9 @@ private function titleTick(){
 	public function handlePacket($address, $port, $payload){
 		try{
 			if(strlen($payload) > 2 and substr($payload, 0, 2) === "\xfe\xfd" and $this->queryHandler instanceof QueryHandler){
+				$ev = new RawPacketReceiveEvent($address, $port, $payload);
```

Comment:
The way you have added this here will only allow raw packet handling by plugins if it's a Query packet if I am reading this correctly. That doesn't seem particularly useful.